code/index.html

```diff
@@ -2,29 +2,88 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+    
+    <!-- HEADER -->
+   
+    <header>  
+
+      <img class="hero-image" src="./images/sunrise.jpg" alt="sunrise" />
+      <h1 class="hero-headline">Adventure trips</h1>
+    </header>
+
+    <!-- CONTENT-->  
+    <div class="content-container">
 
     <!-- Signup form -->
+    <section class="signup-form">
+
+      <h2 class="h2-headline">Log in here</h2>
```

Comment:
Try to come up with as descriptive class names as you can. Maybe just headline is enough here? 😇